install.sh

```diff
@@ -992,15 +992,16 @@ ohai "Downloading and installing Homebrew..."
       execute "${MKDIR[@]}" "${HOMEBREW_CORE}"
       cd "${HOMEBREW_CORE}" >/dev/null || return
 
-      execute "${USABLE_GIT}" "-c" "init.defaultBranch=master" "init" "--quiet"
+      execute "${USABLE_GIT}" "init" "--quiet"
```

Comment:
See https://github.com/Homebrew/install/pull/990#discussion_r2166326492

```suggestion
      execute "${USABLE_GIT}" "-c" "init.defaultBranch=main" "init" "--quiet"
```